Cannot find module `../../lib/Navbar`

I'm not sure what the deal is here - there doesn't seem to be a `lib` folder for `docs/src/NavMain.js` to require from. Am I missing something? 

```sh
09:54 ~/src/react-bootstrap/docs $ npm install
npm WARN package.json react-bootstrap-docs@0.0.0 No repository field.
09:54 ~/src/react-bootstrap/docs $ npm run build

> react-bootstrap-docs@0.0.0 build /Users/richard/src/react-bootstrap/docs
> node build.js && browserify client.js | uglifyjs -cm 2>/dev/null > ./assets/bundle.js

module.js:338
    throw err;
          ^
Error: Cannot find module '../../lib/Navbar'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/richard/src/react-bootstrap/docs/src/NavMain.js:5:14)
    at Module._compile (module.js:460:26)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/richard/src/react-bootstrap/docs/node_modules/node-jsx/index.js:26:12)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

--- a/docs/README.md
+++ b/docs/README.md
@@ -16,7 +16,7 @@ $ cd react-bootstrap/docs
 $ npm install
 ```
 
-You must have previously run `npm install` from the main directory. 
+You must have previously run `grunt build` and `npm install` from the main directory. 
 
 ### Instructions
 

